stories/Breadcrumbs.stories.tsx

```diff
@@ -1,15 +1,15 @@
 import { Story, Meta } from '@storybook/react'
```

Comment:
all `stories/*.stories.tsx` were changed like so:
 - import the component `as <Name>Component`
 - set the story to be `<Name>`

When the only export from a story is the same name as the exported title, it hoists up the display.